app/templates/components/paper-dialog.hbs

```diff
@@ -5,7 +5,7 @@
     fixed=(unless parent true)
     class="md-dialog-backdrop"
     onClick=(action "outsideClicked")}}
-  {{#paper-dialog-container outsideClicked=(action "outsideClicked")}}
+  {{#paper-dialog-container class=(readonly dialogContainerClasses) outsideClicked=(action "outsideClicked")}}
```

Comment:
I think that `class` would be a better API than `dialogContainerClasses`, no?
Since `paper-dialog` is a tagless component, we can just transport the passed in classes to `paper-dialog-container`.